pallets/parachain-staking/src/lib.rs

```diff
@@ -1434,10 +2302,10 @@ pub mod pallet {
 		CandidateBondedMore(T::AccountId, BalanceOf<T>, BalanceOf<T>),
 		/// Candidate, Amount, New Bond
 		CandidateBondedLess(T::AccountId, BalanceOf<T>, BalanceOf<T>),
-		/// Round Offline, Candidate
-		CandidateWentOffline(RoundIndex, T::AccountId),
-		/// Round Online, Candidate
-		CandidateBackOnline(RoundIndex, T::AccountId),
+		/// Candidate
+		CandidateWentOffline(T::AccountId),
+		/// Candidate
+		CandidateBackOnline(T::AccountId),
```

Comment:
Why remove `RoundIndex`?

Reply:
It is an unnecessary GET, we will always know the round in which the event is emitted because we know the block it happened...

Reply:
As a client receiving this event, wouldn't I either need to track which round I'm in or make an extra query to ask for it?

Reply:
As @librelois pointed out, this change reduces blockspace slightly. I no longer have a strong opinion about this...

Reply:
I think that in the absolute, the blocks must contain the strictly necessary and sufficient data, the content of the blocks does not have for role to simplify the life of the users, it is the role of the indexers like subsquid, subscan, etc.